### ISSUE TITLE
How is accuracy calculated when building a model?

### ISSUE THREAD

**Issue by mongoose54** [NVIDIA/DIGITS#305]
So I have a two sets: Training and Testing. I use AlexNet to build my model. The same question applies to other models.

At the end of the model building I see an accuracy of ~90% (see attached figure). Is this accuracy measured on the test set? I assume not because when I put the test set to be classified its accuracy is really bad (~3%). Shouldn't the accuracy be measured on the test set?
![screenshot from 2015-09-14 11 41 04](https://cloud.githubusercontent.com/assets/277391/9854208/96237104-5ad5-11e5-9deb-f42fa5e05f9c.png)

**Comment by lukeyeager** [comment]
The legend shows you which dataset is being used. You're getting one output from the training set - `loss (train)` - and two outputs from the validation set - `loss (val)` and `accuracy (val)`.

If your trained model isn't giving the same performance, then you probably aren't preparing the data properly. See #163.
